hydra/jobsets.nix

```diff
@@ -0,0 +1,65 @@
+{ prs }:
+
+let
+  pkgs = import ./nixpkgs {};
```

Comment:
When I try to run this, I get:

```
hydra [master] ✗ nixops deploy -d riskbook-hydra
error: getting status of '/Users/jgt/Sites/minimal-hs-nix-aws/hydra/nixpkgs': No such file or directory
(use '--show-trace' to show detailed location information)
```

Reply:
So, I added nixpkgs also to `hydra.json` that way it will be embedded correctly.